iron-overlay-manager.html

```diff
@@ -38,11 +38,10 @@
      */
     this._backdropElement = null;
 
-    // Enable document-wide tap recognizer.
-    Polymer.Gestures.add(document, 'tap', null);
-    // We should be using only 'tap', but this would be a breaking change.
-    var tapEvent = ('ontouchstart' in window) ? 'tap' : 'click';
-    document.addEventListener(tapEvent, this._onCaptureClick.bind(this), true);
+    // Listen to mousedown or touchstart to be sure to be the first to capture
+    // clicks outside the overlay.
+    var clickEvent = ('ontouchstart' in window) ? 'touchstart' : 'mousedown';
+    document.addEventListener(clickEvent, this._onCaptureClick.bind(this), true);
```

Comment:
At the very least, it will behave as it was before, when we were listening for either `tap` or `click`, not both..
I checked the jsbin of the reported bug on Android & iOs http://jsbin.com/gajadi/2/edit?html,output and it can't be reproduced on the device. I suspect that's because of `Polymer.Gestures.add(document, 'tap', null);` (which I removed here, since strictly speaking `iron-overlays` don't need it anymore). In a generic Polymer app though this gesture will be enabled by other elements that rely on it.